application/agent_loop/executor.py

```
@@ -0,0 +1,299 @@
+        return last_obs.get("action") == "complete"
+
+    def init_spec(self, task: str) -> Any:
+        """初始化 Spec"""
+        logger.info("=== Initializing Spec ===")
+        return self.spec_generator.generate(task)
+
+    def load_existing_manifest(self) -> Any:
+        """加载现有 Manifest"""
+        try:
+            return self.manifest_manager.load()
+        except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)